README.md

```diff
@@ -32,3 +32,110 @@ Out of the working groups, we anticipate recommendations on governance that will
+- APAC & EMEA DevRel Found Community Call
+
+These calls serve two main purposes:
+
+1. A neutral and safe space to discuss topics important to the community, share ideas, and ask questions aligned with the mission of the DevRel Foundation. If you have a story to share, the recommended process is to open a new issue via GitHub Discussions (more info below) with the topic in mind. This allows the call moderators to bring these topics into sessions, initiate community discussions, and reach a consensus, potentially leading to a concrete action item (e.g., should this become a working group or be included in the foundation, etc.).
+2. Open Office Hours to reach community consensus on the foundation's next phases and to address questions and blockers on DevRel Challenges and Frequently Asked Questions.
+
+<details>
+  <summary>
+    <strong> 📚 How to File Proposals and Discussion Topics </strong>
+  </summary>
+
+The community can file proposals via the DRF Governance repo by [opening a new entry in the GitHub Discussions Forum](https://github.com/DevRel-Foundation/governance/discussions/new/choose).
+
+Also, whether you are new to the DRF community calls or a regular participant, you can add your discussion topics as a comment in one of the [meeting note discussions](https://github.com/DevRel-Foundation/governance/discussions/categories/meeting-notes) or bring them up during the call.
+
+People can also share DevRel use cases, best practices, frameworks, and more by [opening a new Idea in GitHub Discussions](https://github.com/devrel-foundation/governance/discussions/new?category=ideas), which serves as the Foundation’s public forum. 
+
+We are always looking for 5-10 minute presentations in future community calls to spark discussion with attendees and asynchronously.
+
+</details>
+<details>
+  <summary>
+    <strong> 📚 Roles in the DRF Community Calls</strong>
+  </summary>
+
+Roles include facilitator, moderator, scribe, and crier. 
+
+Anyone can volunteer to take on any of these roles for upcoming community calls. People can volunteer to lead in one of these roles during the next call by contacting steering-committee@dev-rel.org
+
+
+| **Role**| **Responsibilities**|**Notes**|**Recommended Experience**|
+|---------|---------------------|---------------|---------------|
+|**Moderator**| (1) Leads the discussion by going through agenda items, ensuring each topic receives the time it needs, and keeping conversations focused AND (2) Opens the call by inviting participants to add their names to the agenda, make group introductions and introduce the concept of Chatham House Rules, LF Antitrust Policies and the DRF Code of Conduct  | Volunteering as a Moderator is a great way to develop facilitation skills and play an essential role in shaping community discussions. The Moderator must be on the call while the Crier can skip the call | This person attended some previous community calls and is familiarized with the format |
```

Comment:
Do we have preferred links for Chatham House Rules and LF Antitrust Policies?

https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md

Reply:
CTA/link about how to volunteer to be a moderator?